src/handlers/v1/markets/[chainId]/[marketId]/leverage/index.ts

```diff
@@ -81,53 +80,51 @@ export const v1GetLeverageQuotationRoute: Route<GetLeverageQuotationRouteParams>
       }
       const leverageUSD = new BigNumberJS(amount).times(leverageCollateral.assetPrice);
 
-      // 1. new balancer flash loan logics and append loan logic
+      // 1. get the quotation for swaping the base token into amount of leverage token.
+      const quotation = await apisdk.protocols.paraswapv5.getSwapTokenQuotation(chainId, {
+        tokenIn: baseToken,
+        output: { token: leverageToken, amount: amount },
+        slippage,
+      });
+      const borrowAmount = quotation.input.amount;
```

Comment:
I understand that we assume we're using a flash loan service without fee, but we might need to separate the flash loan borrow amount and flash loan repay amount (which should be identical to the compound borrow amount) in case the fee isn't 0.

Reply:
Great point! We need to go back and make changes to Protocolink. We should provide a quotation for the flash loan fee to address the situation where using different flash loans might result in insufficient funds for repayment.

For now, keep it as it is. @zodahu, what do you think?